index.html

```diff
@@ -5971,7 +5994,9 @@ <h3>Extension and Import</h3>
     <p>
     The <code>tm:extends</code> feature only permits inheriting all definitions of one <a>Thing Model</a>. In many use cases, however,
      it is desired only to import pieces of definitions of one or more existing <a>Thing Models</a>. 
-     For doing this, the <code>tm:ref</code> term is introduced that provides the location of an existing (sub-)definition that should be reused.
+     <span class="rfc2119-assertion" id="tm-tmRef-usecase">
+     For import pieces of definitions of one or more existing <a>Thing Models</a>, the <code>tm:ref</code> term is introduced that provides the location of an existing (sub-)definition that SHOULD be reused.
```

Comment:
Should be "For importing" or "For the purpose of importing"

Reply:
fixed